python/dgl/graphbolt/internal/sample_utils.py

```diff
@@ -361,7 +367,7 @@ def compact_csc_format(
                 csc_format.indices
             ), "The last element of indptr should be the same as the length of indices."
             src_type, _, dst_type = etype_str_to_tuple(etype)
-            assert len(dst_nodes[dst_type]) + 1 == len(
+            assert len(dst_nodes.get(dst_type, [])) + 1 == len(
```

Comment:
Is this correct? if the dst_type is not in dst_nodes will the len of the csc_format.indptr equals to 1?

Reply:
Yes. The default of cscformat.indptr in sampled subgraph is torch.tensor([0]).